packages/flutter/lib/src/material/text_form_field.dart

```diff
@@ -60,6 +63,7 @@ import 'theme.dart';
 ///    integration.
 ///  * [InputDecorator], which shows the labels and other visual elements that
 ///    surround the actual text editing widget.
+///  * <https://flutter.dev/docs/cookbook/forms/text-field-changes#2-use-a-texteditingcontroller>
```

Comment:
Linking to a cookbook example is a good idea. I suggest making it a bit more readable:
```suggestion
///  * Learn how to use a [TextEditingController] in one of our [cookbook recipe]s.(https://flutter.dev/docs/cookbook/forms/text-field-changes#2-use-a-texteditingcontroller)
```

Reply:
done